community_images/argocd/big-bang-ironbank/image.yml

```diff
@@ -0,0 +1,45 @@
+name: argocd-bigbang-ib
+official_name: Argo CD Big-Bang Iron Bank
+official_website: https://argoproj.github.io/cd/
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/big-bang/argocd
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/big-bang%2Fargocd
+source_image_readme: https://repo1.dso.mil/dsop/big-bang/argocd/-/blob/development/README.md
+rf_docker_link: rapidfort/argocd-bigbang-ib
+image_workflow_name: argocd_bigbang_ironbank
+github_location: argocd/bigbang-ironbank
```

Comment:
Github location must match the `<image-name>/<provider-name>` format of directories. So, it  must be `argocd/big-bang-ironbank`.

>Here `-` is missing in bigbang